### ISSUE TITLE
required input - unable to set custom isEmpty message

### ISSUE THREAD

**Issue by j-schumann** [j-schumann/vrok-lib#2]
When using an inputSpecification using required=true we could use a notEmpty validator in the chain to set our custom error message when the input was not set:

```
return [
            'name'     => $this->getName(),
            'required' => true,
            'filters'  => [
                ['name' => 'Zend\Filter\StringTrim'],
            ],
            'validators' => [
                [
                    'name'                   => 'Zend\Validator\NotEmpty',
                    'break_chain_on_failure' => true,
                    'options'                => [
                        'messages' => [
                            \Zend\Validator\NotEmpty::IS_EMPTY => 'custom message',
                        ],
                    ],
                ],
                // further validators
            ],
        ];
```

InputFilter would then use this validator to get the error message. But now Zend\InputFilter\Input uses prepareRequiredValidationFailureMessage() the default NotEmpty::IS_EMPTY message without any option to customize this.
There is a pull request open to fix this https://github.com/zendframework/zend-inputfilter/pull/73 but it is not merged yet while other solutions are discussed...

**Comment by j-schumann** [comment]
seems fixed on ZF side since 2.6.1 - 2016-04-07 -> fix here

**Comment by j-schumann** [comment]
fixed in CSRF-Element
